src/libqof/qof/qofbackend-p.h

```diff
@@ -354,6 +354,8 @@ extern "C"
  */
 void qof_backend_register_provider (/*@ only @*/ QofBackendProvider *);
 
+void qof_backend_unregister_provider (QofBackendProvider *);
```

Comment:
Uh, the backends register their providers at load time and unregister_all_providers is a cleanup at application shutdown to free them all. That's changed in c++-backend (the vector gets unique_ptrs that free themselves) so better to leave it alone in this change to reduce merge conflicts.